lib/tasks/clean.rake

```diff
@@ -16,4 +16,10 @@ namespace :clean do
   task :dead_queues do
     CleanDeadQueuesJob.set(queue: :low).perform_later
   end
+
+  desc "Clean failed PendingUpload Resources."
+  task :pending_uploads, [:dry_run] => [:environment] do |_t, args|
+    args.with_defaults(dry_run: false)
+    CleanPendingUploadsJob.set(queue: :low).perform_later(dry_run: args[:dry_run].present?)
```

Comment:
Perhaps the job should take a logger and dry-run should do perform_now with a stdout logger.